fields/class-swp-acf-color-palette-v5.php

```diff
@@ -176,7 +176,7 @@ function render_field( $field ) {
 			?>
 			<div class="acf-color-palette">
 				<?php acf_hidden_input( $hidden_input ); ?>
-                <input type="text" <?php echo acf_esc_attr( $text_input ); ?> data-alpha="true">
```

Comment:
This is not the right way to operate because `acf_esc_attr` will output a `class` attribute too. That will be better to modify the variable `$text_input` like that: `$text_input['class'] .= ' color-picker';`.